### PR TITLE
Added filters to @@bookings

### PR DESCRIPTION
**Pull request by pcdummy**
Added all missing filters from @@orders to @@bookings.

See b.p.shop issue [#67](https://github.com/bluedynamics/bda.plone.shop/issues/67)

Signed-off-by: Rene Jochum <rene@jochums.at>

### CODE REVIEW

File: src/bda/plone/orders/browser/bookings.py [hunk 1]
```diff
@@ -108,47 +115,109 @@ class BookingsTable(BrowserView):
...
         )
-        filter = to_date(request=self.request)
-        return filter
+        filter_widgets += to_date(request=self.request)
+
+        # vendor areas of current user
+        vendors = vendors_form_vocab()
+        vendor_selector = None
+        # vendor selection, include if more than one vendor
+        if len(vendors) > 2:
```
Comment (rnixx) [inline src/bda/plone/orders/browser/bookings.py]: We might should skip vendor filter for "bookings in context" and "orders in context" or reduce the vocabulary to all vendors found in the specific context if we want to keep the filter.

Comment (thet) [inline src/bda/plone/orders/browser/bookings.py]: reducing the filters to what's available would be awesome anyways, even if not shown "in context" and also for @@orders...
don't know about the effort for doing this, but IF it's easy and fast to implement....
A "clear all filters" button would then be good.

**Comment by pcdummy** [comment]
This is ready for a review.

**Comment by thet** [comment]
Looks good to me!

**Pull request merged**